python/assistant-stream/src/assistant_stream/create_run.py

```
@@ -30,6 +34,17 @@ async def add_tool_call(
         self.add_stream(stream)
         return controller
 
+    async def add_tool_result(self, tool_call_id: str, result: Any) -> None:
```

Ruff (0.8.2)

Dynamically typed expressions (typing.Any) are disallowed in `result`

(ANN401)